core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -1278,6 +1278,30 @@ public String toString() {
               + "used when Zookeeper is enabled, since Zookeeper already stores the master "
               + "addresses.")
           .build();
+  public static final PropertyKey MASTER_FILE_ACCESS_TIME_JOURNAL_FLUSH_INTERVAL =
+      new Builder(Name.MASTER_FILE_ACCESS_TIME_JOURNAL_FLUSH_INTERVAL)
+          .setDefaultValue("1h")
+          .setDescription("The minimum interval between files access time update journal entries "
+              + "get flushed. Setting it to a non-positive value will make the the journal update "
+              + "synchronous.")
+          .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
+          .setScope(Scope.MASTER)
+          .build();
+  public static final PropertyKey MASTER_FILE_ACCESS_TIME_UPDATE_PRECISION =
+      new Builder(Name.MASTER_FILE_ACCESS_TIME_UPDATE_PRECISION)
+          .setDefaultValue("1d")
+          .setDescription("The file last access time is precise up to this value. Setting it to"
+              + "a non-positive value will update last access time on every file access operation.")
```

Comment:
can you add some implications of short vs long precisions?

Reply:
Updated.